juriscraper/pacer/email.py

```diff
@@ -222,22 +225,38 @@ def _get_description(self):
                     if des_part == "Notice will be electronically mailed to:":
                         break
                     description = description + des_part
-                return clean_string(description)
+                description = clean_string(description)
+                if description:
+                    return description
         else:
             path = '//strong[contains(., "Docket Text:")]/following-sibling::'
             node = self.tree.xpath(f"{path}font[1]/b//text()")
             if len(node):
                 for des_part in node:
                     description = description + des_part
-                return clean_string(description)
+                description = clean_string(description)
+                if description:
+                    return description
 
             node = self.tree.xpath(f"{path}b[1]/span//text()")
             if len(node):
                 for des_part in node:
                     description = description + des_part
-                return clean_string(description)
+                description = clean_string(description)
+                if description:
+                    return description
+
+            node = self.tree.xpath(f"{path}text()")
+            if len(node):
+                for des_part in node:
+                    description = description + des_part
+                description = clean_string(description)
+                if description:
+                    return description
```

Comment:
With this kind of repetitive code, I usually like to do a little loop:

```
for path in possible_paths:
    node = self.tree.xpath(path)
    ...
```
Makes it more clear that the same code is being run over and over. I think you could even just put the appellate logic into the same loop without much trouble too, but I haven't checked that carefully.